src/plugins/messageservices/imap/imapclient.cpp

```diff
@@ -94,132 +95,12 @@ class DataFlushedWrapper : public QMailMessageBufferFlushCallback
 
 namespace {
 
-    QString decodeModifiedBase64(QString in)
```

Comment:
This is moved to "imapfoldernamecoding.cpp" for consistency with 'encode' new stuff